services/mesh-forwarder/tasks

```diff
@@ -0,0 +1,387 @@
+  # https://www.openssl.org/docs/manmaster/man1/req.html
+  openssl genrsa -out ${keys_file_name}.key 2048
+  openssl req -new -key ${keys_file_name}.key -out ${keys_file_name}.csr -config ${CERTIFICATES_DIR}/csr.conf
+}
+
+###########
+## TASKS ##
+###########
+
+command="$1"
+case "${command}" in
+  _csr)
+    check_env
+    if [ -z $ODS_CODE ]; then
+      echo "Please specify ODS_CODE"
+      exit 4
+    fi
+    fqdn="$ODS_CODE.nems.$NHS_ENVIRONMENT.non-prod.patient-deductions.nhs.uk"
+    generate_csr "$ODS_CODE-nems-$NHS_ENVIRONMENT" $fqdn
+    echo "The CSR has to be sent together with endpoint registration. Use the form below or ask endpoint admin to register it"
+    echo "https://digital.nhs.uk/forms/combined-endpoint-and-service-registration-request"
+    ;;
+    csr)
+     ./tasks _csr
+      ;;
+  _csr_nems_prod)
+    if [ -z $ODS_CODE ]; then
+      echo "Please specify ODS_CODE"
+      exit 4
+    fi
+    fqdn="nems-$ODS_CODE.nhsdigital.nhs.uk"
+    generate_csr "$ODS_CODE-nems-prod" $fqdn
+    ;;
+   csr_nems_prod)
+    ./tasks _csr_nems_prod
+    ;;
+
+  _csr_mesh)
+    check_env
+    if [ -z $ODS_CODE  ]; then
+      echo "Please specify ODS_CODE "
+      exit 4
+    fi
+    generate_mesh_fqdn
+    generate_csr "$ODS_CODE-mesh-$NHS_ENVIRONMENT" $MESH_FQDN
+    echo "For Non prod the CSR has to be added to mesh mailbox request form"
+    echo "https://digital.nhs.uk/services/message-exchange-for-social-care-and-health-mesh/messaging-exchange-for-social-care-and-health-apply-for-a-mailbox"
+    echo "Details on how to fill out the form is on confluence"
+    echo "https://gpitbjss.atlassian.net/wiki/spaces/TW/pages/11604721782/Requesting+a+MESH+mailbox"
+    ;;
+   csr_mesh)
+    ./tasks _csr_mesh
+    ;;
+
+  _get_authorisation_token)
+    _get_authorisation_token
+    ;;
+  get_authorisation_token)
+    check_env
+    ./tasks _get_authorisation_token
+    ;;
+  _create_nems_subscription_and_update_ssm)
+    _assume_environment_role $NHS_ENVIRONMENT
+    set_nems_env
+    python3 create_subscription_and_update_ssm.py
+    echo Successfully processed subscription
+    ;;
+  create_nems_subscription_and_update_ssm)
+    ./tasks _create_nems_subscription_and_update_ssm
+    ;;
+  _test_nems_subscriber)
+    get_local_tkw_subscriptions_env
+    pushd nems-subscriber && pip3 install -r requirements.txt && python3 -m pytest --capture=no
+    ;;
+  test_unit)
+    ./tasks _test_nems_subscriber
+    ;;
+  _create_nems_subscription)
+    set_nems_env
+    python3 create_subscription.py
+    ;;
+  create_nems_subscription)
+    ./tasks _create_nems_subscription
+    ;;
+  _read_nems_subscription)
+    set_nems_env
+    python3 read_subscription.py
+    ;;
+  read_nems_subscription)
+    ./tasks _read_nems_subscription
+    ;;
+  _delete_nems_subscription)
+    set_nems_env
+    python3 delete_subscription.py
+    ;;
+  delete_nems_subscription)
```

Comment:
We can clean up this tasks file massively. The non-underscored functions I believe were written originally so that they shell into Dojo and then call the underscored functions. Now we've moved away from Dojo, we can get rid of the duplicated functions.